exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebUserConnection.java

```diff
@@ -64,6 +66,8 @@
 
   public final Set<String> columns = Sets.newLinkedHashSet();
 
+  public final ArrayList<String> metadata = Lists.newArrayList();
```

Comment:
Guava suggests to use the ArrayList constructor directly, taking advantage of the new "diamond" syntax:
https://google.github.io/guava/releases/23.0/api/docs/com/google/common/collect/Lists.html
Thus `public final List<String> metadata = new ArrayList<>();` will be better.

Reply:
Done